packages/twenty-front/src/modules/workspace/components/WorkspaceProviderEffect.tsx

```diff
@@ -15,17 +18,64 @@ export const WorkspaceProviderEffect = () => {
     workspacePublicDataState,
   );
 
+  const [lastAuthenticateWorkspace, setLastAuthenticateWorkspace] =
+    useRecoilState(lastAuthenticateWorkspaceState);
+
+  const {
+    redirectToHome,
+    getWorkspaceSubdomain,
+    redirectToWorkspace,
+    isTwentyHomePage,
+  } = useUrlManager();
+
+  const isMultiWorkspaceEnabled = useRecoilValue(isMultiWorkspaceEnabledState);
+
   useGetPublicWorkspaceDataBySubdomainQuery({
```

Comment:
I think we need to send the subdomain from the frontend, the back cannot guess it

Reply:
maybe not actually